sync/class.jetpack-sync-client.php

```diff
@@ -1,7 +1,8 @@
 <?php
+require_once dirname(__FILE__) . '/class.jetpack-sync-deflate-codec.php';
 
 class Jetpack_Sync_Client {
-	private $sync_queue = array();
+	public $sync_queue = array();
```

Comment:
I don't think we should expose this property publicly as it's very likely to change, particularly as we introduce persistence etc.